supportedBackends/local/src/test/resources/application.conf

```diff
@@ -0,0 +1,10 @@
+local-config {
+  root = "local-cromwell-executions"
+  filesystems = {
+    local {
+      localization: [
+        "hard-link", "soft-link", "copy"
+      ]
+    }
+  }
+}
```

Comment:
It's the default conf used in the tests. When you do ConfigFactory.load() this is what you get.